Assignments/Program.cs

```diff
@@ -1,3 +1,55 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T19 - The Factorial Number
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
+
+namespace Spark;
+public class T19 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Nth Factorial Computer:-" + "\x1B[0m");
+      List<int> testCases = new () { 0, 1, 2, 3, 5, 8, 10, 12, 15, 16 };
+      Console.WriteLine ("Test Cases:");
+      for (int i = 0; i < testCases.Count; i++) {
+         int t = testCases[i];

```

Comment:
't' variable is not used anywhere except 'PrintResults'. So, you can directly call it there.
`PrintResult (testCases[i]);`